stylelint.config.js

```diff
@@ -16,5 +17,27 @@ module.exports = {
 				ignorePseudoClasses: [...stylelintConfig.rules['selector-pseudo-class-no-unknown'][1].ignorePseudoClasses, 'global'],
 			},
 		],
+		'csstools/use-logical': [
+			'always',
+			{
+				severity: 'warning',
+				// Only lint LTR-RTL properties for now
+				except: [
+					// Position properties
+					'top',
+					'bottom',
+					// Position properties with directional suffixes
+					/-top$/,
+					/-bottom$/,
+					// Size properties
+					'width',
+					'max-width',
+					'min-width',
+					'height',
+					'max-height',
+					'min-height',
```

Comment:
There is a logical alternative? :smile:

Reply:
https://github.com/csstools/stylelint-use-logical?tab=readme-ov-file#property-and-value-mapping